main.py

```diff
@@ -73,7 +73,32 @@
 
     await bot.process_commands(message)
 
+#start CuckTim Section
+#replaces "Cuck" with Rotating Light Emoji, meant to piss off Tim
+@bot.event
+async def on_message(message):
+    if not message.author.bot and "cuck" in message.content.lower():
+        CuckString = message.content.replace('cuck', ':rotating_light:')
+        newString = re.compile("cuck", re.IGNORECASE)
+        await bot.send_message(message.channel, newString.sub('cuck', cuckString))
+    elif (str(ctx.message.author) == "RacistJudicata#8536"):
```

Comment:
This won't do what you think it is supposed to do.

If anyone (that isn't a bot) says "cuck", it will be caught by the first conditional, even if RJ says it. `elif` here will only be hit if the `if` condition resolves to `false`. 

What will happen howerver, is that **every** comment by RJ (that doesn't contain 'cuck') will hit this code. Meaning Don will reply to everything Tim does with "Stop it, Tim".

While hilarious, it's probably not what we want to do.